src/Sylius/Behat/Page/Admin/Product/IndexPerTaxonPage.php

```diff
@@ -19,6 +19,15 @@
 
 class IndexPerTaxonPage extends CrudIndexPage implements IndexPerTaxonPageInterface
 {
+    public function getProductPosition(string $productName): int
+    {
+        if (!$this->hasElement('product_taxon_position', ['%name%' => $productName])) {
+            return -1;
```

Comment:
This is hacky (even more so since we can set position `-1`), either remove this if and operate on the `ElementNotFoundException` or create a method `isProductOnPage` or smth similar